src/ResizeSensor/ResizeSensor.ts

```diff
@@ -1,28 +1,41 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import _debounce from 'lodash/debounce';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import _debounce = require('lodash/debounce');
 import { props, t, ReactChildren } from '../utils';
-import _ResizeSensor from 'css-element-queries/src/ResizeSensor';
+import { ResizeSensor as _ResizeSensor } from 'css-element-queries';
 
 export const Props = {
   children: ReactChildren,
   onResize: t.Function,
   debounce: t.maybe(t.Integer)
 };
 
+export type ResizeSensorProps = {
+  /** content */
+  children: JSX.Element,
+  /** called when a resize event is captured */
+  onResize: () => void,
+  /** callback delay (milliseconds) */
+  debounce?: number
+};
+
+interface ResizeSensorElement extends Element {
```

Comment:
this is to convince TS that it's ok to access the `resizedAttached` property on the DOM node.